Tests/test_SeqIO.py

```diff
@@ -133,18 +133,32 @@ def test_gzip_fastq(self):
         """Testing FASTQ with gzip."""
         with gzip.open("Quality/example.fastq.gz", "rt") as handle:
             self.assertEqual(3, len(list(SeqIO.parse(handle, "fastq"))))
+        with gzip.open("Quality/example.fastq.gz") as handle:
+            with self.assertRaisesRegex(
+                ValueError, "Is this handle in binary mode not text mode"
+            ):
+                list(SeqIO.parse(handle, "fastq"))
 
     def test_gzip_fasta(self):
         """Testing FASTA with gzip."""
         with gzip.open("Fasta/flowers.pro.gz", "rt") as handle:
             self.assertEqual(3, len(list(SeqIO.parse(handle, "fasta"))))
+        with gzip.open("Fasta/flowers.pro.gz") as handle:
+            with self.assertRaisesRegex(
+                ValueError, "Is this handle in binary mode not text mode"
+            ):
+                list(SeqIO.parse(handle, "fasta"))
 
     def test_gzip_genbank(self):
         """Testing GenBank with gzip."""
         # BGZG files are still GZIP files
         with gzip.open("GenBank/cor6_6.gb.bgz", "rt") as handle:
             self.assertEqual(6, len(list(SeqIO.parse(handle, "gb"))))
-
```

Comment:
Flake8 will complain, two blank lines between classes. This is just one of the ways adopting black will make life easier.